infrastructure/Cdk/Program.cs

```diff
@@ -0,0 +1,17 @@
+using Amazon.CDK;
+
+namespace Cdk;
+
+public sealed class Program
+{
+	public static void Main(string[] args)
+	{
+		var app = new App();
+		_ = new RaidmateStack(app, "RaidmateStack", new StackProps
+		{
+			StackName = "RaidmateBackendStack"

```

Comment:
I think this should have a suffix/prefix for env? (prod, stage, devname, etc)

Reply:
#8